### PR TITLE
Add docs for Ember.

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
…i adapter

### CODE REVIEW

File: docs/howto/ember-and-json-api.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  keyForRelationship: function(rawKey) {
+    return underscore(rawKey);
+  }
+});
+
+```
+
+## Including Nested Resources
+
+Previously, `store.find` and `store.findRecord` did not any query params.
```
Comment (beauby) [inline docs/howto/ember-and-json-api.md]: Missing word "did not any query params."

Comment (NullVoxPopuli) [inline docs/howto/ember-and-json-api.md]: updated

File: docs/howto/ember-and-json-api.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+render json: @post, include: params[:include]
+```
+
+If you want to use `include` on a collection, you'd write something like this:
+
+```javascript
+store.query('post', { include: 'comments' });
+```
+
+which will egnerate the path `/posts?include='comments'`
```
Comment (beauby) [inline docs/howto/ember-and-json-api.md]: > egnerate

**Comment by beauby** [comment]
Caught one typo, LGTM other than that (although I'm not an Ember user, so I'd appreciate it if somebody with Ember experience could review this one).

**Comment by beauby** [comment]
Could you squash?

**Comment by NullVoxPopuli** [comment]
squashed

File: docs/howto/ember-and-json-api.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# How to use JSON API Query Parameters with Ember
```
Comment (bf4) [inline docs/howto/ember-and-json-api.md]: or may just `docs/integrations/ember.md`

Comment (NullVoxPopuli) [inline docs/howto/ember-and-json-api.md]: :+1:

**Pull request merged**